fastlane/lib/fastlane/actions/get_provisioning_profile.rb

```diff
@@ -15,7 +15,6 @@ def self.run(values)
         require 'credentials_manager/appfile_config'
 
         Sigh.config = values # we already have the finished config
-        Sigh.config[:api_key] ||= Actions.lane_context[SharedValues::APP_STORE_CONNECT_API_KEY]
```

Comment:
Not needed anymore because `Sigh` will look for existing `Spaceship::ConnectAPI.client` now and use that